### PR TITLE
added some basic data cleaning and scaling

### PR DESCRIPTION
**Pull request by The-Pavel**
In this branch doing some basic data preprocessing:

* Data cleaning
* Scaling (standard)
* Adding datetime features (DoW, month, hour)

### CODE REVIEW

**Comment by phaberman** [comment]
NICE CODE BRO!!!!

File: my-project-api/main.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Doing some awesome data preprocessing...")
```
Comment (The-Pavel) [inline my-project-api/main.py:1]: Maybe you can do more than just printing??????

Comment (phaberman) [reply my-project-api/main.py:1]: Looks good to me!

Comment (phaberman) [reply my-project-api/main.py:1]: Approved!

**Review by phaberman — APPROVED** [review]

**Pull request merged**